backend/infrahub/core/manager.py

```diff
@@ -295,6 +299,7 @@ async def count(
             at=at,
             partial_match=partial_match,
             branch_agnostic=branch_agnostic,
+            order={"disable": True},
```

Comment:
Instead of passing a dict around I think it would be best to create a Dataclass or a Pydantic model for order.